### DISCUSSION TITLE
getMinRateByPair

### DISCUSSION THREAD

**Discussion by jurgenBE** [bancorprotocol/carbon-sdk#140] [Q&A] [answered]
on the site : https://docs.carbondefi.xyz/developer-guides/carbon-defi-sdk/sdk-functions/getminratebypair and then use the @bancor/carbon-sdk, but i don't see an example for this, only the import {
  PayableOverrides,
  TradeActionBNStr,
  TokenPair,
  MatchActionBNStr,
  StrategyUpdate,
  EncodedStrategyBNStr,
} from '@bancor/carbon-sdk'; etc.. can you give an example how to use this for example ?

**Comment by barakman** [comment, marked as answer]
> on the site : https://docs.carbondefi.xyz/developer-guides/carbon-defi-sdk/sdk-functions/getminratebypair and then use the @bancor/carbon-sdk, but i don't see an example for this, only the import { PayableOverrides, TradeActionBNStr, TokenPair, MatchActionBNStr, StrategyUpdate, EncodedStrategyBNStr, } from '@bancor/carbon-sdk'; etc.. can you give an example how to use this for example ?

```
const minRate: string = await getMinRateByPair(sourceTokenAddress, targetTokenAddress);
```
